src/libostree/libostree-devel.sym

```diff
@@ -18,10 +18,6 @@
 ***/
 
 /* Add new symbols here.  Release commits should copy this section into -released.sym. */
-LIBOSTREE_2017.8 {
-global:
-  ostree_validate_remote_name;
-} LIBOSTREE_2017.7;
```

Comment:
It looks like you may also have to line up the hunk for the next release as well.